server.js

```diff
@@ -20,4 +23,42 @@ app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+const getAllMessages = (req, res) => res.status(200).json({ data: messages });
+
+const postMessage = (req, res) => {
+  const { from, text } = req.body;
+
+  from.trim() === "" ? res.status(200).json("Fill from field please!") : null;
+  text.trim() === "" ? res.status(200).json("Fill text field please!") : null;
+  const newId = messages.length !== 0 ? messages[messages.length - 1].id + 1 : 0;
+  const newMessage = {
+    id: newId,
+    from: from,
+    text: text
+  }
+  messages.push(newMessage);
+  res.status(200).json("Your message added!");
+};
+
+const getOneMessage = (req, res) => {
+  const { id } = req.params;
+  const message = messages.find(eachMessage => eachMessage.id === Number(id));
+  !message ? res.status(200).json("Please enter a valid Id") :
+    res.status(200).json({ data: message });
+};
+
+const deleteMessage = (req, res) => {
+  const index = messages.findIndex(eachMessage =>
+    eachMessage.id === Number(req.params.id));
+  index === -1 ? res.status(404).json("Your message not found!") : messages.splice(index, 1) && res.status(200).json("Your message deleted!");
```

Comment:
This syntax looks kinda weird (`x` && `y` where x and y are statements to be executed rather than boolean comparisons) - from some brief testing it seems like it works, however I'd advise against it just as it makes it less obvious what the code is doing, and could be interpreted as a boolean comparison instead.